src/ucs/datastruct/bitmap.h

```diff
@@ -359,18 +363,60 @@ typedef uint64_t ucs_bitmap_word_t;
     }
 
 
+/**
+ * Find the index of the first bit set to 1 in a given bitmap, starting from
+ * a particular index (excluding it)
+ *
+ * @param bitmap      Look for the first bit in this bitmap
+ * @param nun_words   Number of words in the bitmsp
+ * @param start_index The first bit to look from
+ */
+int _ucs_bitmap_ffs(void *bitmap, size_t num_words, size_t start_index)
```

Comment:
1. remove _
2. can we move to C file / or declare this as static inline, or u get a link err of dup symbol

Reply:
also can it be `const ucs_bitmap_word_t *bitmap`

Reply:
Done. Though, I'm not sure we want to expose this function. Bitmap words are an internal implementation detail, and I won't like users to use it directly.